core/src/main/java/org/mskcc/cbio/oncokb/model/Evidence.java

```diff
@@ -173,7 +173,27 @@ public boolean equals(Object obj) {
         return true;
     }
 
-
+    public Evidence CloneEvidence(){

```

Comment:
Maybe write a copy constructor instead of CloneEvidence. And one small style issue: Java method should start to a lower case letter.